projects/addon-doc/src/components/language-switcher/language-switcher.component.html

```diff
@@ -0,0 +1,37 @@
+<tui-select
+    [tuiTextfieldSize]="size"
+    [tuiDropdownLimitWidth]="limitWidth"
+    [valueContent]="content"
+    [formControl]="language"
+>
+    Language
+    <input
+        tuiTextfield
+        placeholder="Language"
+    />
+
+    <ng-template tuiDataList>
+        <tui-data-list>
+            <button
+                *ngFor="let name of names"
+                tuiOption
+                [value]="name"
+                [textContent]="flags.get(name)"
```

Comment:
This way you overwrite internal structure of `tuiOption` component.

Reply:
fixed